src/content_cao.cpp

```diff
@@ -645,7 +645,15 @@ void GenericCAO::addToScene(ITextureSource *tsrc)
 		}
 		m_wield_meshnode = new WieldMeshSceneNode(
 			RenderingEngine::get_scene_manager(), -1);
-		m_wield_meshnode->setItem(item, m_client);
+		m_wield_meshnode->setItem(item, m_client,
+			(m_prop.visual == "wielditem"));
+		
+		// Does wield_image need to be checked or not
+		if (m_prop.visual == "wielditem") {
```

Comment:
This code was replaced by L648 & L649. Forgot to delete perhaps?

Reply:
Apparently. That, or I just didn't notice (likely).